pommerman/envs/v3.py

```diff
@@ -0,0 +1,101 @@
+"""The Pommerman v3 Environment, which implements Sudden Death mode.
+
+The game ends when any agent dies. In FFA, all other agents win. In any of the team
+modes, the team with the most remaining players wins. If all players die at the same
+time or the same number of players die on each team, the game ends in a tie. All
+players/teams lose if the game ends without anyone dying.
+"""
+from .. import constants
+from .. import forward_model
+from .. import utility
+from . import v0
+
+
+class Pomme(v0.Pomme):
+    metadata = {
+        'render.modes': ['human', 'rgb_array']
+    }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _get_done(self):
```

Comment:
By doing this, we're breaking the compatibility with training models, e.g. train_with_tensorforce. Don't skip using the forward model / reuse code. 

Instead, get the result from the forward model and change it if the sudden death has occurred.